Example/src/LayoutReanimation/DefaultAnimations.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+import { View, Text, Button, StyleSheet, ScrollView, TouchableOpacity, TouchableWithoutFeedback } from 'react-native';
+import Animated, { 
+  AnimatedLayout, 
+  SlideInRight, 
+  SlideOutRight,
+  SlideInUp, 
+  SlideInDown, 
+  SlideOutUp, 
+  SlideOutDown,
+  FadeIn, 
+  FadeOut, 
+  SlideOutLeft, 
+  SlideInLeft,
+  ZoomIn,
+  ZoomInRotate,
+  ZoomOut,
+  ZoomOutRotate,
+  StretchInX,
+  StretchInY,
+  StretchOutX,
+  StretchOutY,
+  FlipInXUp,
+  FlipInYLeft,
+  FlipInXDown,
+  FlipInYRight,
+  FlipInEasyX,
+  FlipInEasyY,
+  FlipOutXUp,
+  FlipOutYLeft,
+  FlipOutXDown,
+  FlipOutYRight,
+  FlipOutEasyX,
+  FlipOutEasyY,
+  ZoomInRight,
+  ZoomInLeft,
+  ZoomInUp,
+  ZoomInDown,
+  ZoomInEasyUp,
+  ZoomInEasyDown,
+  BounceIn,
+  BounceOut,
+  BounceInDown,
+  BounceInUp
```

Comment:
I wonder if we can group those without any performance penalty, such big import list looks kinda funny 😄